charts/pathfinder-rpc-server/README.md

```diff
@@ -0,0 +1,68 @@
+### Pathfinder rpc server chart
+
+This chart includes the component in the table below in order to  isolate the pathfinder rpc server installation.
+
+For a detailed description please refer to [land-local](https://github.com/CirclesUBI/land-local) 
+ Repo                                                                     | Service               | Description                                                                              |
+|--------------------------------------------------------------------------|-----------------------|------------------------------------------------------------------------------------------|
+| [blockchain-indexer](https://github.com/CirclesUBI/blockchain-indexer)   | blockchain-indexer    | Indexes circles related on-chain events and writes them to a postgres db                 |
+| [pathfinder2](https://github.com/CirclesUBI/pathfinder2)                 | pathfinder2           | Finds transitive payment paths between untrusted users                                   | 
+| [pathfinder2-updater](https://github.com/CirclesUBI/pathfinder2-updater) | pathfinder2-updater   | Updates the 'pathfinder2' with data from the 'blockchain-indexer'                        | 
+| [pathfinder-proxy](https://github.com/CirclesUBI/pathfinder-proxy)       | pathfinder-proxy      | Maintains statistics, load balances and filters requests to the 'pathfinder2'            |
+
+
+### Pre-requisities
+
+Create the following secrets in your cluster:
+    - blockchain-indexer-connection-string: connection string for the indexer to the db
+    - pathfinder-updater-connection-string: connection string for the pathfinder to the db
+    - indexer-db-secret: `password` for the `doadmin` user and `postgres-password`  for the `postgres` user 
+
+Manage your secrets with your prefered tool. 
+
+### Currents caveats and usage 
+
+#### Caveats
+- The indexer db is empty and without proper schemas and there is not an indexer-db-init. It only exists for local development with ganache. 
+- To syncronise the indexer fully at the start it requires a lot resources and its own rpc node. There is still the option to install your own rpc node if you want to go down this road and avoid the following workaround. However, for quicker sync and  resources saving  we recommend the workaround. 
+
+#### Workaround 
```

Comment:
Instead of loading the dump into a database of an indexer that is already running, you could drain the indexer nodes to 0 replicas and create an empty database to load into. Only when you're finished, you could resaturate the indexer.

Reply:
yeah I tried this but still doesnt work cause I cannot connec to the db with 
`kubectl -n pathfinder exec -i pod/pathfinder-rpc-server-postgresql-0 -- sh -c 'pg_restore  -p 5432 -U ${DB_USER} -v -d ${DB}' < indexer.dump` (including and leaving out `-h localhost`.  I cannot enter the password and get `pg_restore: error: connection to server on socket "/tmp/.s.PGSQL.5432" failed: FATAL:  password authentication failed for user "postgres"` same with the forward-port and the others